.github/workflows/databend-release.yml

```diff
@@ -45,8 +45,13 @@ jobs:
         run: |
           brew install gnu-tar
           sudo /usr/sbin/purge
-          gtar -C ./target/${{ steps.target.outputs.target }}/release -czvf databend-${{ steps.get_version.outputs.VERSION }}-${{ steps.target.outputs.target }}.tar.gz databend-query databend-meta
-          gtar -C ./target/${{ steps.target.outputs.target }}/release -czvf databend-tools-${{ steps.get_version.outputs.VERSION }}-${{ steps.target.outputs.target }}.tar.gz databend-metactl
+          mkdir release/${{ steps.target.outputs.target }} release/${{ steps.target.outputs.target }}/bin release/${{ steps.target.outputs.target }}/configs
+          cp ./target/${{ steps.target.outputs.target }}/release/databend-* release/${{ steps.target.outputs.target }}/bin/
+          rm -f release/${{ steps.target.outputs.target }}/bin/*.d
+          cp ./scripts/ci/deploy/config/databend-query-node-1.toml release/${{ steps.target.outputs.target }}/configs/databend-query.toml
```

Comment:
Is this config suitable for the user's example? Do we need to provide a separate example?

Reply:
These two configuration files basically cover the normal configuration needs and I think we can start by using it as a basic configuration example.

Another benefit may be that we use these configurations in our tests, so subsequent maintenance will be more timely.

As the deployment documentation is further refined, we will eventually provide some more sensible configurations.